pkg/mhooks/event.go

```diff
@@ -0,0 +1,47 @@
+package mhooks
+
+import "github.com/moovfinancial/moov-go/pkg/moov"
+
+type EventType string
+
+const (
```

Comment:
Note that I omitted the `wildcard (*)` type, that can be used when configuring a webhook subscription, but not an actual event type we'd send to users